2996_reflection/reflection.md

```diff
@@ -3472,12 +3468,6 @@ Change paragraphs 6-9 of [dcl.init.general]{.sref} [No changes are necessary for
 * [7.2]{.pnum} If T is an array type, [...]
 * [7.*]{.pnum} [If `T` is `std::meta::info`, the object is zero-initialized.]{.addu}
 * [7.3]{.pnum} Otherwise, no initialization is performed.
-
-[8]{.pnum} A class type `T` is *const-default-constructible* if [`T` is `std::meta::info`,]{.addu} default-initialization of `T` would invoke a user-provided constructor of T (not inherited from a base class)[,]{.addu} or if
-
-* [8.1]{.pnum} [...]
-
-[9]{.pnum} To value-initialize an object of type T means: [...]
```

Comment:
Yep, but `std::meta::info` isn't a class type, and I don't think we need anything here: Default initialization (defined above, paragraph 7) has a special case for `std::meta::info`. That should cover `constexpr meta::info v;`, right?

Reply:
Yeah but there's still this point: http://eel.is/c++draft/dcl.dcl#dcl.init.general-8.sentence-2:

> If a program calls for the default-initialization of an object of a const-qualified type T, T shall be a const-default-constructible class type or array thereof.

So basically need some way to carve out `meta::info`.

Reply:
Ahh, I see - I think you're right, but we need to massage it a bit. This change came up in CWG and there was confusion because it sounds like `std::meta::info` is "a class type". Don't think we got past that confusion to what was being achieved by the change.

The word "constructible" brings to mind class types, so I think it makes more sense to update the sentence you cited rather than the definition of _const-default-constructible_ types. I've updated with a possible direction; let me know if you think it works.